Samsung/A53/res/values/configs.xml

```diff
@@ -0,0 +1,512 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer-array name="config_screenBrightnessBacklight">
```

Comment:
Sounds useless to me...? This array doesn't fix anything, does it?

Reply:
Tbh yes it doesn't.. It was there in arrays.xml so I just added them.